### PR TITLE
DAOS-8326 test: update dm_large_file and dm_large_posix yaml files

### PR DESCRIPTION
**Pull request by dsikich**
The dm_large_file test is intermittently failing with a DER_NOSPACE
error when the pool is being created. This updates the yaml to
instead use percentages for scm and nvme so that the test will
not try to create a pool that is too big for the CI cluster.

The dm_obj_large_posix test is intermittently failing with a
timeout. The cont_clone portion alone was taking about 5
minutes on successful runs, and the timeout limit only had
about 21 seconds left. This updates the yaml to give the
cont_clone test two more minutes to complete.

Test-tag: pr datamover

Signed-off-by: Danielle Sikich <danielle.sikich@intel.com>

### CODE REVIEW

**Review by saurabhtandan — DISMISSED** [review]
LGTM

**Review by daltonbohning — COMMENTED** [review]
Looks like the dm_dst_create test is failing because of the duns_resolve_path changes

**Comment by dsikich** [comment]
> Looks like the dm_dst_create test is failing because of the duns_resolve_path changes

@daltonbohning yeah, this PR will have to wait until there is a fix merged and I can merge/rebase.

File: src/tests/ftest/datamover/dm_large_file.yaml
Comment (mchaarawi) [inline src/tests/ftest/datamover/dm_large_file.yaml:24]: i see the same issue in dm_large_dir.yaml

Comment (dsikich) [reply src/tests/ftest/datamover/dm_large_file.yaml:24]: @mchaarawi I updated this to just use percentages in the yaml for both.

File: src/tests/ftest/datamover/dm_large_dir.yaml
Comment (daltonbohning) [inline src/tests/ftest/datamover/dm_large_dir.yaml:25]: For large_dir this is okay, but realistically the datasize of the large_dir test is very small.
```
50k files * 4096 bytes ~= 200MB
```
Some other tests use a similar dataset with a much smaller pool size:
https://github.com/daos-stack/daos/blob/9025d7cfef2f366557c95df5f3a20a49c62a0668/src/tests/ftest/datamover/dm_obj_large_posix.yaml#L26-L27
https://github.com/daos-stack/daos/blob/9025d7cfef2f366557c95df5f3a20a49c62a0668/src/tests/ftest/datamover/dm_serial_large_posix.yaml#L24-L25

**Review by daltonbohning — APPROVED** [review]
Code-wise this look okay, but it looks like some builds are failing (due to some Jenkins issues?)

**Comment by dsikich** [comment]
> Code-wise this look okay, but it looks like some builds are failing (due to some Jenkins issues?)

@daltonbohning yeah I guess Jenkins was paused for issues and haven't heard an update yet.

File: src/tests/ftest/datamover/dm_large_dir.yaml
Comment (dsikich) [reply src/tests/ftest/datamover/dm_large_dir.yaml:25]: > For large_dir this is okay, but realistically the datasize of the large_dir test is very small.
> 
> ```
> 50k files * 4096 bytes ~= 200MB
> ```
> 
> Some other tests use a similar dataset with a much smaller pool size:
> 
> https://github.com/daos-stack/daos/blob/9025d7cfef2f366557c95df5f3a20a49c62a0668/src/tests/ftest/datamover/dm_obj_large_posix.yaml#L26-L27
> 
> 
> https://github.com/daos-stack/daos/blob/9025d7cfef2f366557c95df5f3a20a49c62a0668/src/tests/ftest/datamover/dm_serial_large_posix.yaml#L24-L25

@daltonbohning yeah but the pool being created in the test looked pretty large in dm_large_file.yaml.

Comment (dsikich) [reply src/tests/ftest/datamover/dm_large_dir.yaml:25]: @daltonbohning yeah I see, could also just use smaller pool size like those tests I suppose.

Comment (daltonbohning) [reply src/tests/ftest/datamover/dm_large_dir.yaml:25]: @dsikich For `large_file` it matters. But for these `large_dir` tests I don't think it really matters either way, as long as the pool size is reasonably large. Probably not necessary to push another commit, anyway.

**Review by mchaarawi — APPROVED** [review]

**Pull request merged**
